src/test/java/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookMultibranchTriggerTest.java

```diff
@@ -0,0 +1,79 @@
+package com.atlassian.bitbucket.jenkins.internal.trigger;
+
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketPluginConfiguration;
+import com.atlassian.bitbucket.jenkins.internal.config.BitbucketServerConfiguration;
+import com.atlassian.bitbucket.jenkins.internal.credentials.GlobalCredentialsProvider;
+import com.atlassian.bitbucket.jenkins.internal.scm.BitbucketSCMRepository;
+import com.atlassian.bitbucket.jenkins.internal.scm.BitbucketSCMSource;
+import hudson.model.FreeStyleProject;
+import hudson.model.ItemGroup;
+import jenkins.branch.MultiBranchProject;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.multibranch.WorkflowMultiBranchProject;
+import org.junit.Before;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static java.util.Optional.of;
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class BitbucketWebhookMultibranchTriggerTest {
```

Comment:
Since this seems to be testing the `Descriptor` in `BitbucketWebhookMultibranchTrigger`, should we rename the test to reflect that?
```suggestion
public class BitbucketWebhookMultibranchTriggerDescriptorTest {
```

Reply:
Hmm actually I think we want one or two tests covering the start method here- will add those.
I'm not sure if this is a rule but I feel like we tend to separate out descriptor and decribable classes when we have a lot of tests and there's a separation in terms of functionality (SCM tests are a good example of this, as the descriptors are mostly just verifying correct data). I think in this case, given they mirror functionality so closely, it makes sense to keep the tests together under the parent class name.

Reply:
Hmm on reflection start is really just filtering and calling, won't worry about a test.